### PR TITLE
B9lab platform updates part I

### PR DESCRIPTION
**Pull request by coldice**
This adds the first part of the B9lab platform updates, preparing the platform functionality and config for the following content updates. We have added [a separate readme](https://github.com/cosmos/sdk-tutorials/tree/b9lab-platform-updates/b9lab-content#information-for-cosmos) to outline the process, steps and further details, as discussed with @okwme.

This PR:
* Integrates the H5P standalone viewer, wrapped in a vuepress component.
* Extends the code block component, adding a Multi Code Box (CodeGroup).
* Adds Expansion panel and Highlight Box (info, tip, warning and further reading) component (Note: the styling of the highlight box icons/colors will be updated).
* Prepares the content file structure in `b9lab-content`.
* Adds the content outline to `.vuepress/config.js` (currently deactivated).
* Adds a [feature-test](https://deploy-preview-790--sdk-tutorials.netlify.app/feature-test/) page which demonstrates the usage of these new components.

### CODE REVIEW

**Comment by coldice** [comment]
[BLOCKING] Need to check possible license conflict on the included h5p-fontawesome (4.5) pack - we do own a perpetual pro license.

File: .vuepress/public/h5p/test-arithmetic-quiz/H5P.ArithmeticQuiz-1.1/css/arithmetic-quiz.css [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+  box-shadow: none;
+}
+.h5p-baq .h5p-joubelui-button:focus {
+  outline: 2px dashed #fff;
+  outline-offset: 4px;
+}
+.h5p-baq-game {
+  display: table-cell;
+}
+/* Don not show progress when igniting or when showing results*/
```
[the remark this answers is not in the document]
Comment (coldice) [reply .vuepress/public/h5p/test-arithmetic-quiz/H5P.ArithmeticQuiz-1.1/css/arithmetic-quiz.css:40]: These files are auto-generated by the H5P Editor, I have pushed the fix upstream https://github.com/h5p/h5p-arithmetic-quiz/pull/52

File: .vuepress/public/h5p/test-arithmetic-quiz/H5P.ArithmeticQuiz-1.1/language/af.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
+{
```
[the remark this answers is not in the document]
Comment (coldice) [reply .vuepress/public/h5p/test-arithmetic-quiz/H5P.ArithmeticQuiz-1.1/language/af.json:1]: The H5P module supports multiple languages and comes with base support (menus, messages) for many, but the actual content (eg. quiz questions and answers) will initially only be available in english. To support more languages, we would need to add a language selection (logic) to the platform and think about the overall content translation, but yes: This module is already ready.

File: b9lab-content/3-main-concepts/11-base-app.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# BaseApp
```
[the remark this answers is not in the document]
Comment (coldice) [reply b9lab-content/3-main-concepts/11-base-app.md:1]: If it is fine for you, we would like to adjust the file names at a later point, as outlined in [my reply here](https://github.com/cosmos/sdk-tutorials/pull/790/#issuecomment-907362753).

**Review by okwme — APPROVED** [review]

**Pull request merged**